### ISSUE TITLE
Ignoring max extracted limit

### ISSUE THREAD

**Issue by kam193** [CybercentreCanada/assemblyline#149]
**Is your feature request related to a problem? Please describe.**
I have a dynamic analyze service working in parallel to the extraction and static analyze (if it matters). Sometimes extraction from an archive exhausts the maximum extracted files limit, what has two consequences for the dynamic service: 1) it cannot extract additional files (expected), 2) it cannot add dumped network connection traffic (and here is my issue) as extracted file for further analyze.

**Describe the solution you'd like**
I would like to be able to set a flag in `add_extracted`, that a given file should be extracted regardless of the exhausted limit. I'd like to analyze the traffic, even if the limit was reached, and drop other files.

**Describe alternatives you've considered**
1. Automated resubmit for dynamic analyze - would work, but I want to keep static and dynamic results together.
2. Hacking the `add_extracted` method - I haven't tried, but looks like temporary overriding the `task.max_extracted` may solve my issue. But maybe there is a second check later.
3. Leaving it as is - the number of issues isn't very high, so I could go along with it.

**Additional context**

**Comment by cccs-rs** [comment]
Have you considered raising the `max_extracted` limit in your [submission params](https://cybercentrecanada.github.io/assemblyline4_docs/odm/models/submission/#submissionparams) or perhaps setting the global configuration for all submissions to have a higher `max_extracted` limit ([submission.default_max_extracted](https://cybercentrecanada.github.io/assemblyline4_docs/odm/models/config/#submission))?

**Comment by cccs-rs** [comment]
Would there be a way to safelist those not relevant files (assuming they are indeed safe files)?

Extract is configured to refer to the Safelist when it's adding extracted files: if the extracted file has been deemed safe by AL, then it doesn't append the file to the results.

**Comment by kam193** [comment]
Not really, they are not the same :( But I try with higher limit, looks like I missed that ingest flow has lower default as submission

**Comment by cccs-rs** [comment]
Correct, you'll want to update `core.ingester.default_max_extracted`.

**Comment by cccs-rs** [comment]
Any updates on this issue?

**Comment by kam193** [comment]
So the higher limit has made the problem less important, but sometimes it's still not enough. I can live with that - I still find the option to ignore the limit useful, but I wouldn't say it's a big problem for me anymore. Thanks for asking :)

**Comment by cccs-rs** [comment]
Will close this issue for now since we would like to maintain an upper bound on the max files extracted in a submission, and the current solution of raising that limit via configuration/request sounds acceptable for now.
